pages/pixel-ui/avatar.mdx

```diff
@@ -0,0 +1,22 @@
+import MethodList from "/components/method_list";
+
+# Avatar
+## Methods
+The following vgui methods are accessible via the `PANEL` table:

```

Comment:
This should say the following:
The following methods are accessible through the `Avatar` element table: